simpleLooping/test_binaryListToString.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+from binaryListToString import binaryListToString
+
+class TestBinaryListToString(unittest.TestCase):
+    def test_listBinaryInput(self):
+        self.assertEqual(binaryListToString([0, 1, 1, 0, 0, 1]), "100110")
+
+    def test_notListBinaryInput(self):
+        self.assertEqual(binaryListToString([2, 4, 6]), "-1")
```

Comment:
“-1” is not equal with -1. One is string and one is integer. Did you run the unit test? Did it return the correct result as you expected?